server/src/actions/createUser.ts

```diff
@@ -0,0 +1,18 @@
+// Uploads a new document in the Users collection.
+import { doc, setDoc } from '@firebase/firestore'
+import { users } from '../utilities/firebaseInit'
+
+export const createUser = async (displayName: string, userId: string, avatarUrl: string, lat: number, lon: number, geohash: string) => {
```

Comment:
Possibly making the parameters for createUser into an interface on top of the function for better readability?

Reply:
The parameters come directly from the User type, which is defined as an interface. I've already created an issue for this to be refactored, and currently I'm having a hard time doing so while keeping it readable.

Reply:
Ok sounds good!